fuzz/fuzz.dict

```diff
@@ -0,0 +1,257 @@
+"__loadScript"
```

Comment:
How is this dict created? If automated, is it possible to check-in the script that does so?

Reply:
The names were collected with a JS script, I'll upload it in the next round.